python/ray/_private/ray_logging/constants.py

```diff
@@ -45,6 +45,7 @@ class LogKey(str, Enum):
     ACTOR_NAME = "actor_name"
     TASK_NAME = "task_name"
     TASK_FUNCTION_NAME = "task_func_name"
+    PID = "pid"
 
     # Logger built-in context
```

Comment:
Python LogRecord has a built-in attr: `process` which is pid so we should use that: https://docs.python.org/3/library/logging.html#logrecord-attributes

Reply:
Do you mean renaming this key to match the built-in attr name `PROCESS = "process"`? [Here](https://github.com/ray-project/ray/pull/55176/files#diff-6b77e25edf7973caec6a63223bbe8c8220054037e1a092c3614f2bd8e7370299R10), I am using the value of the built-in `record.process`